conf/layer.conf

```diff
@@ -2,3 +2,9 @@
 
 # We have a conf and classes directory, append to BBPATH
 BBPATH .= ":${LAYERDIR}"
+
+BBFILE_COLLECTIONS += "cyclonedx"
+BBFILE_PATTERN_cyclonedx = "^${LAYERDIR}/"
+
+LAYERDEPENDS_cyclonedx = "core"
+LAYERSERIES_COMPAT_cyclonedx = "styhead"
```

Comment:
Didn't we should use `LAYERSERIES_COMPAT_cyclonedx = "scarthgap styhead"` instead ? As far as I know, you said it was compatible with it

Reply:
It is (currently, but with the merge of https://github.com/savoirfairelinux/meta-cyclonedx/pull/11 it wouldn't be anymore) but the idea is to have the main branch of this repository align with yoctos master branch and the scarthgap branch in this repository align with yoctos scarthgap branch. Thus, each branch should only define compatibility with the appropriate release branch.

Reply:
oh okay, that's make sense !
Will take care of this in afternoon then